README.md

```diff
@@ -185,7 +186,6 @@ group.background {
 group.utility {
     // Run on utility queue, in parallel to the previous block
 }
-group.wait()
```

Comment:
Do you think `.wait()` is an bad use, or why is it removed. I think `.notify { }` could work as an alternative, not a replacement.